trade-finance-manager-api/src/v1/api.js

```diff
@@ -442,6 +442,27 @@ const findFacilitiesByDealId = async (dealId) => {
   }
 };
 
+const getAcknowledgedCompletedAmendments = async (dealId) => {
```

Comment:
```suggestion
const getFullySubmittedAmendments = async (dealId) => {
```

To keep this function name future proof please.